zkevm-circuits/src/tx_circuit.rs

```diff
@@ -2367,7 +2600,30 @@ impl<F: Field> TxCircuitConfig<F> {
         let (access_list_address_size, access_list_storage_key_size) =
             access_list_size(&tx.access_list);
 
-        // fixed_rows of a tx
+        // Only bytes from L2 txs are accumulated for chunk bytes hash
+        let is_chunk_bytes = (tx.tx_type != TxType::L1Msg) && !tx.caller_address.is_zero();
```

Comment:
It's better to add a new function `is_real_l2_tx` for type `Transaction` and put these two conditions into it.